docs/GettingStarted.md

```diff
@@ -12,8 +12,40 @@ your system, so that you can build apps with it right away. If you already
 have React Native installed, you can skip ahead to the
 [Tutorial](docs/tutorial.html).
 
-The instructions are a bit different depending on your development operating system, and whether you want to start developing for iOS or Android. If you
-want to develop for both iOS and Android, that's fine - you just have to pick
+## Quick Start
+
+[Create React Native App](https://github.com/react-community/create-react-native-app) is the easiest way to start building a new React Native application. It allows you to start a project without installing or configuring any native build tools.
```

Comment:
This phrasing is meant to mirror [Create React App's in the React docs](https://facebook.github.io/react/docs/installation.html#creating-a-single-page-application), but is fungible as far as I'm concerned.